seb_openedx/edxapp_wrapper/backends/get_course_module_h_v1.py

```diff
@@ -1,7 +1,19 @@
 """ Backend abstraction """
+from xmodule.modulestore import ModuleStoreEnum  # pylint: disable=import-error
 from xmodule.modulestore.django import modulestore  # pylint: disable=import-error
 
 
 def get_course_module(course_key, depth=0):
     """ get_course_module backend """
     return modulestore().get_course(course_key, depth=0)
+
+
+def modulestore_update_item(course_key, course_module, user_id):
+    """
+    update_item backend.
+    Updates both the draft and published branches.
+    """
+    with modulestore().bulk_operations(course_key):
+        with modulestore().branch_setting(ModuleStoreEnum.Branch.draft_preferred):
```

Comment:
Should we consider an edge case when `draft` and `published` could be out of sync? Maybe catching an exception here?

Reply:
For that exact reason I'm not updating `draft` and then publishing it. Here we are merely updating both branches individually from whichever point they currently are

Reply:
ok